docs/configuration/large_language_model.md

```diff
@@ -88,21 +88,66 @@ ludwig train ...
 
 ## Input Features
 
-Currently, the LLM model type only supports a single input feature of type `text`.
+Because of the way LLMs work, they accept exactly one input, which is some amount of `text`. 
```

Comment:
nit: `of type text` instead of `some amount of text`? I think it also gets the point across of it needing to be a text type feature in a Ludwig config

Reply:
Done